src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -261,7 +261,7 @@ public bool NotUsingHttps(string url)
                     waitingTasks.Add(requester.ExecuteAsync(new WaitForRaftIndexCommand(index), context));
                 }
 
-                await Task.WhenAll(waitingTasks);
+                await Task.WhenAny(waitingTasks);
```

Comment:
this code?
https://github.com/karmeli87/ravendb/blob/fc55be770a1bcdfd649e4bfaa887d8d1c81dc362/src/Raven.Server/Web/System/AdminDatabasesHandler.cs#L270